packages/toolkit/src/node/process.js

```diff
@@ -60,7 +63,9 @@ export const startProcess = async ({
       if (data.includes(readyOutput)) {
         resolve({
           output,
-          kill: () => killProcessTree(subprocess, { logger }),
+          kill: () => {
```

Comment:
We need to return `killProcessTree` here so that we can `await` and get notified when this function has finished. Let's revert this

Reply:
Fixed in https://github.com/aragon/aragon-cli/pull/1072